tests/spread/core22/remote-build/snaps/build-for-with-archs/snapcraft.yaml

```diff
@@ -0,0 +1,20 @@
+name: build-for-with-archs-core22
```

Comment:
I was hitting the 40-character limit for the snap names, hence I renamed all these tests to simpler and shorter names. Sorry for the noise!